app/src/main/java/com/douglasbruce/tasky/core/data/database/di/DatabaseModule.kt

```diff
@@ -0,0 +1,25 @@
+package com.douglasbruce.tasky.core.data.database.di
+
+import android.content.Context
+import androidx.room.Room
+import com.douglasbruce.tasky.core.data.database.TaskyDatabase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DatabaseModule {
+    @Provides
+    @Singleton
+    fun providesTaskyDatabase(
+        @ApplicationContext context: Context,
+    ): TaskyDatabase = Room.databaseBuilder(
+        context,
+        TaskyDatabase::class.java,
+        "tasky_database"
+    ).build()
+}
```

Comment:
I'd probably bundle this with the DAOs in the previous module, since a DB module won't really contain anything else other than the DB instance right? Maybe a type converter later, but that's it